scripts/jenkins/qa_openstack.sh

```diff
@@ -446,18 +446,45 @@ if [ -e /etc/tempest/tempest.conf ]; then
     fi
 
     pushd /var/lib/openstack-tempest-test/
+
+    # Handle OpenStack release specific blacklisting of known to fail tests
+    blacklist_args=""
+    case "${cloudsource}" in
+    (openstackrocky)
+        # TODO(fmccarthy): Remove once we have addressed issues causing
+        # failures for the neutron_tempest_plugin tests (SCRD-8681)
+        # If neutron_tempest_plugin is installed then blacklist known
+        # to fail tests
+        if rpm -q python-neutron-tempest-plugin > /dev/null 2>&1; then
+            _blacklist_file=$(mktemp /tmp/neutron_tempest_plugin_blacklist.XXXXXXXX)
+            cat - | tee ${_blacklist_file} << __EOF__
+# Blacklist the tests matching the pattern: neutron_tempest_plugin\.api\.admin\.test_tag\.Tag(Filter|)(QosPolicy|Trunk)TestJSON
+#neutron_tempest_plugin.api.admin.test_tag.TagFilterQosPolicyTestJSON.test_filter_qos_policy_tags
+id-c2f9a6ae-2529-4cb9-a44b-b16f8ba27832
+#neutron_tempest_plugin.api.admin.test_tag.TagQosPolicyTestJSON.test_qos_policy_tags
+id-e9bac15e-c8bc-4317-8295-4bf1d8d522b8
+#neutron_tempest_plugin.api.admin.test_tag.TagTrunkTestJSON.test_trunk_tags
+id-4c63708b-c4c3-407c-8101-7a9593882f5f
+#neutron_tempest_plugin.api.admin.test_tag.TagFilterTrunkTestJSON.test_filter_trunk_tags
+id-3fb3ca3a-8e3a-4565-ba73-16413d445e25
+__EOF__
```

Comment:
Just a nitpick style preference but I would avoid python-isms like the dunderscore and the leading _ for private variables

Reply:
Use of the double __ in the shell here document delimiter __EOF__ has nothing to do with Python, and is a pattern that I (and many others) have frequently used over last last 28+ years of VAX/VMS scripting and Unix/Linux/POSIX shell programming... Similarly for my habit of using _ to indicate local variables.

Reply:
Okay, it's not consistent with the style of the rest of the script though, that was my main motivation for pointing it out and the other style inconsistencies. Ultimately not critical of course.